deploy/cd-namespace/chart/values.yaml

```diff
@@ -37,22 +37,19 @@ aquaRegistry: ''
 # Leave empty when not using Aqua.
 aquaUsername: ''
 
-# PVC (used for the pipeline workspace)
-# Storage provisioner. On AWS backed clusters, use 'kubernetes.io/aws-ebs'.
-storageProvisioner: 'kubernetes.io/aws-ebs'
-# Storage class. On AWS backed clusters, use 'gp2'.
-storageClassName: 'gp2'
-# Storage size. Defaults to 2Gi unless set explicitly here.
-storageSize: '5Gi'
-
 # Cluster
 # URL (including scheme) of the OpenShift Web Console.
 consoleUrl: 'http://example.com'
 
 # Webhook Interceptor
 interceptor:
-  # API host (no scheme) of the OpenShift/K8s API.
-  apiHost: 'openshift.default.svc.cluster.local'
+  # PVC (used for the pipeline workspace)
+  # Storage provisioner. On AWS backed clusters, use 'kubernetes.io/aws-ebs'.
+  storageProvisioner: 'kubernetes.io/aws-ebs'
+  # Storage class. On AWS backed clusters, use 'gp2'.
```

Comment:
is 'standard' same as 'gp2' in this case?

Reply:
I don't think so. `standard` is used on the KinD cluster, `gp2` on AWS.

Reply:
asked because we set `standard` as default, but looks good in any case